scripts/thrust_target_to_motion_point_transcriber.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import rospy
+
+from iarc7_msgs.msg import TwistStampedArray
+from iarc7_msgs.msg import MotionPointStampedArray, MotionPointStamped
+
+def callback(msg):
+    motion_points = MotionPointStampedArray()
+
+    for twist in msg.twists:
+        motion_point = MotionPointStamped()
+        motion_point.header = twist.header
+        motion_point.motion_point.twist = twist.twist
+        motion_points.motion_points.append(motion_point)
+
+    motion_point_pub.publish(motion_points)
+
+
+if __name__ == '__main__':
```

Comment:
I really don't know how I feel about this node. I think it introduces a layer that can be easily removed by updating HLM to use the motion point message. Also, it can introduce lag between sending a request from HLM to LLM and idk if that is desirable or even acceptable.

Reply:
Also, considering that if this node were to crash, communication between HLM and LLM would be severed. This is obviously a bad situation so if this were to remain, we would need some kind of safety response. At minimum, it should be using `iarc7_safety` to ensure its bonded/running.

Reply:
This node only exists because HLM hasnt been updated. I forsee a lot of discussion on that. How long till HLM is updated? If its a long time the safety response can be added but that requires changes to launch that I wanted to avoid.

This node is rather simple and has a very low chance of crashing.